components/03-modules/modal/modal.config.js

```diff
@@ -14,5 +30,28 @@ module.exports = {
       attributes: 'type="button" aria-label="close modal button, click to close the modal"'
     },
     script: true
-  }
+  },
+  variants: [
+    {
+      name: 'header-footer',
+      context: {
+        header: {
+          html: '<h3 id="myTitle">Save "untitled" document?</h3>',
+        },
+        main: {
+          html: '<div id="myDesc">You have made changes to "untitled.txt" that have not been saved. What do you want to do?</div>'
+        },
+        footer: {
+          html: 'Some text here or via component',
+          content: 'button',
+          contentContext: {
+            text: 'Close',
+            tag: 'button',
+            class: 'button',
+            attributes: 'type="button" aria-label="Close"'
```

Comment:
if a button has text the `aria-label` attribute is not necessary, b/c screen reader reads what's inside the button, so if the text is the same it's useless, please remove